dbt_semantic_interfaces/pyproject.toml

```diff
@@ -20,7 +20,6 @@ pandas = "^1.3.0"
 PyYAML = "^6.0"
 mo-sql-parsing = "^9.328.23003"
 more-itertools = "8.10.0"
-yamllint = "^1.26.3"
```

Comment:
You'll need to run poetry update in both package directories, otherwise we'll technically still ship with this in the package.

Reply:
TIL